index.html

```diff
@@ -249,14 +249,34 @@ <h1>The Registration Process</h1>
 Any method name SHOULD avoid generic terms such as "mymethod" or "registry".
       </li>
       <li>
-If there are copyright, trademark, or any intellectual property rights
-concerns, the addition and use MUST be authorized in writing by the intellectual
-property rights holder under a
-<a href="https://en.wikipedia.org/wiki/Reasonable_and_non-discriminatory_licensing">F/RAND</a>
-license. Examples include DID Methods that use trademarked brand names,
-property names that utilize the titles of copyrighted works, and patented
-technology that would cause the use of the extension to require licensing a
-patent.
+Any DID Method submission MUST not impose any concerns about copyright,
+trademark, or intellectual property rights on the W3C, DID Method
+implementers, or users. To that end, the following apply:
+        <ol>
+          <li>
+DID Method submitters affirm to the best of their knowledge that their 
+submission will not infringe upon or otherwise violate the rights of any third 
+party.
+          </li>
+          <li>
+If there are any known concerns about copyright, trademark, or intellectual
+property rights, the addition and use of a DID Method MUST be authorized in
+writing by the holder of the intellectual property rights under a 
```

Comment:
I *think* `F/RAND` is pronounced `eff-rand`, which calls for `an`. It *might* be pronounced `frand`, which would call for `a`. I couldn't find any recordings of a subject matter expert speaking the term, so this change request is based on my best guess of its pronunciation.
```suggestion
writing by the holder of the intellectual property rights under an
```